templates/patterns/list_item/pattern-list-item.html.twig

```diff
@@ -2,12 +2,31 @@
 /**
  * @file
  * Default item list component.
+ *
+ * @see @ecl/card
  */
 #}
-{% include '@ecl/ec-component-list-item' with {
-  'href': url,
-  'metas': meta,
-  'title': title,
-  'detail': detail,
-  'screen_reader': 'List item'|t
+{% set _infos = {} %}
+{% for infos_item in infos %}
+  {% set _info = {
+    label : infos_item.text,
+    icon : {
+      type : 'general',
+      name : infos_item.icon,
+      path : ecl_icon_path
+    }
+  } %}
+  {% set _infos = _infos|merge([_info]) %}
+{% endfor %}
```

Comment:
As said above, I'd remove the `infos` all together in this PR and move it to a follow up, maybe we will get ECL Twig to provide this by the time we need them.